packages/nextjs/services/database/repositories/users.ts

```diff
@@ -249,3 +247,50 @@ export async function filterValidUserAddresses(addresses: string[]): Promise<str
     .where(inArray(users.userAddress, addresses));
   return rows.map(row => row.userAddress);
 }
+
+const CHALLENGE_NUMBER = 7; // Number of challenges currently possible
```

Comment:
Just making this up right now, but is there a better way for us to get the total number of challenges possible?

Reply:
Yes!!

You can count all the challenges in the database (`challenges` table).

You will have to NOT count:
- challenges with `disabled = TRUE`
- challenges with `autograding = FALSE` (svg nft and multisig are still there)